website/blog/2021-04-08-GAAD-March-Accessibility-Issue-Update.md

```diff
@@ -0,0 +1,75 @@
+---
+title: The GAAD Pledge - March Accessibility Issues Update
+author: Alexandra Marlette
+authorTitle: GAAD Pledge Open Source Accessibility Community Manager for React Native
+authorURL: 'https://twitter.com/alexmarlette'
+authorImageURL: 'https://avatars.githubusercontent.com/u/10052470?s=460&u=7f2304cb929d1de703856717af86324c66728f3a&v=4'
+authorTwitter: alexmarlette
+tags: [announcement]
+---
+
+It has been four weeks since we reached out to the Github community with a thoroughly reviewed gap analysis and list of issues to improve React Native accessibility. With the help of the React Native community, we are already making great progress on improving accessibility. Community members have been helping contributors, reviewing tests, and bringing attention to prior accessibility issues. Since March 8th the community has closed six issues with four pull requests and seven other pull requests are in the pipeline for review.
+
+While this work continues, the React Native and Accessibility teams at Facebook are also busy continuing to evaluate accessibility bugs and issues that were submitted prior to this initiative, to determine if they are already covered by our current gap analysis or if there are additional issues that need to be brought into the project. One issue has already been discovered and moved into the project, four others directly mapped to existing gap analysis issues and two others are expected to be closed by addressing existing issues already in the project that address the root cause of their issue.
```

Comment:
```suggestion
While this work continues, the React Native and Accessibility teams at Facebook are evaluating accessibility bugs and issues that were submitted prior to this initiative, to determine if they are already covered by our current gap analysis or if there are additional issues that need to be brought into the project. One new issue has already been discovered and moved into the project, four others directly mapped to existing issues and two others are expected to be closed by addressing existing issues that address the root cause of their issue.
```

Small nit suggestion to move to active tense and remove a few extra words!

Reply:
Thank you @kacieb!